rust/hyperlane-core/src/chain.rs

```diff
@@ -57,6 +57,21 @@ impl From<&'_ Address> for H160 {
     }
 }
 
+impl From<fuels::tx::ContractId> for Address {
```

Comment:
Is this and the `impl From<Address> for fuels::tx::ContractId` below used anywhere? I tried commenting them out and `cargo build` worked just fine for me

I wouldn't expect these to be necessary - I don't think we'll ever want to go from a 160 bit Address to a 256 bit ContractId or vice versa, so I'd remove

Reply:
It was used at one point. Yes unused now.